src/core/settings.py

```diff
@@ -318,7 +313,7 @@ def _load_api_raml(path):
 
 INGESTION_LOG_NAME = 'ingestion-%s.log' % PROJECT_NAME
 
-LOG_DIR = PROJECT_DIR if ENV == DEV else '/var/log/'
+LOG_DIR = PROJECT_DIR if DEBUG else '/var/log/'
```

Comment:
this looks like a separate flag, which is probably just the log path; suppose I turned on debug in end2end to get more introspection about exceptions, etc; I won't expect logs to change folder?

Reply:
you wouldn't ever turn DEBUG on in end2end. it's meant purely for development

Reply:
I'm not sure DEBUG gives you more introspection about exceptions either

Reply:
The question then is what `DEBUG` actually does then (adding more routes?)
I suggest to continue the move from environment name to flag, and introduce the `log_path` configuration in `app.cfg`; a default of `/var/log` seems the safest for rollout, though a default of `.` will make it easier to use locally.

Reply:
DEBUG comes with Django and is used by Django and is typically expected to be there by third party apps. I'm not going to remove it. The ENV stuff was there since the beginning and it's usage has been slowly eroded until it can be entirely removed.

In lax DEBUG is used to include admin content resources so the /admin interface can be rendered properly while running the development server.

> I suggest to continue the move from environment name to flag, and introduce the log_path configuration in app.cfg; a default of /var/log seems the safest for rollout

I disagree, I think my decision here is best.

Reply:
if I wasn't clear, DEBUG == native, ENV == me, ages ago. The trick with the urls + DEBUG flag is a common one